templates/theories/category_theory.md

```diff
@@ -58,7 +39,7 @@ We use `≅` for isomorphisms.
 ### Functors
 
 We use `⥤` (`\func`) to denote functors, as in `C ⥤ D` for the type of functors from `C` to `D`.
-(Unfortunately `⇒` is reserved in [`logic.relator`](https://github.com/leanprover-community/mathlib/blob/master/src/logic/relator.lean), so we can't use that here.)
+(Unfortunately `⇒` is reserved in [`Mathlib.Logic.Relator`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Logic/Relator.html), so we can't use that here.)
```

Comment:
This is no longer true! This stuff is now in Mathlib and we could easily scope the exotic relator notation.

Reply:
Fixed, thanks.